patches/tModLoader/Terraria/Item_tML.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Terraria.ModLoader.IO;
+
+namespace Terraria
+{
+	public partial class Item : TagSerializable
```

Comment:
No, I definitely don't think we should break the namespace and folder structure duality we have so far.